bot.js

```diff
@@ -123,9 +123,11 @@ if (!process.env.clientId || !process.env.clientSecret) {
   require(__dirname + '/components/plugin_glitch.js')(controller);
 
   var normalizedPath = require("path").join(__dirname, "skills");
-  require("fs").readdirSync(normalizedPath).forEach(function(file) {
-    require("./skills/" + file)(controller);
-  });
+  require("fs").readdirSync(normalizedPath)
+.filter(file => file !== '.DS_Store')
```

Comment:
Indenting is off here. Let’s make sure we aren’t committing white space things like this unless it’s very pertinent.

Reply:
fixed.